File: android/src/main/java/com/jimmydaddy/imagemarker/base/ErrorCode.java
package com.jimmydaddy.imagemarker.base;

public enum ErrorCode {
  INVALID_PARAMS("INVALID_PARAMS"),
  PARAMS_REQUIRED("PARAMS_REQUIRED");

  private final String value;

  private ErrorCode(String value) {
    this.value = value;
  }

  public String getValue() {
    return value;
  }
}

File: android/src/main/java/com/jimmydaddy/imagemarker/base/ErrorCode.kt
package com.jimmydaddy.imagemarker.base

enum class ErrorCode(val value: String) {
  INVALID_PARAMS("INVALID_PARAMS"),
  LOAD_IMAGE_FAILED("LOAD_IMAGE_FAILED"),
  GET_RESOURCE_FAILED("GET_RESOURCE_FAILED"),
  PARAMS_REQUIRED("PARAMS_REQUIRED"),
  NULL_MAP("NULL_MAP");
}
